### ISSUE TITLE
Specify visibility options in config file

### ISSUE THREAD

**Issue by martin-ueding** [edsu/feediverse#44]
The [Mastodon documentation](https://mastodonpy.readthedocs.io/en/stable/) states that one can also pass the `visibility` key, when one creates a Toot:

    'visibility': # Toot visibility ('public', 'unlisted', 'private', or 'direct')

This would enable to have a feed as *unlisted* such that it doesn't show up on in the main timeline. The necessary changes are likely very small.

I could try to do a pull request, but there are already pull requests touching the same lines. This would lead to conflicts. So let's decide whether you would want to have that feature anyway.

**Comment by Wikinaut** [comment]
Important to have.

**Comment by Wikinaut** [comment]
Yes, @martin-ueding , please send a pull-request for this issue, it is urgently needed.

**Comment by martin-ueding** [comment]
So this project is somewhat dead now? At least the original author did not react here. I doubt that they would merge the changes.
